dll/win32/syssetup/wizard.c

```diff
@@ -2968,6 +2970,29 @@ ProcessSetupInf(
     }
 #endif
 
+    *szValue = UNICODE_NULL;
+    pData = szValue;
+
+    GetWindowsDirectoryW(szValue, ARRAYSIZE(szValue));
+    StringCchCatW(szValue, ARRAYSIZE(szValue), L"\\inf");
+
+    if (FAILED(StringCchLengthW(szValue, ARRAYSIZE(szValue), &pcchLength)))
+    {
+        return;
+    }
+
+    pData += pcchLength + 1;
+    pcchLength = (ARRAYSIZE(szValue) - pcchLength - 1);
+    StringCchCopyW(pData, pcchLength, pSetupData->SourcePath);
+
+    if (FAILED(StringCchLengthW(pData, ARRAYSIZE(szValue), &pcchLength)))
```

Comment:
Hm... Should this ARRAYSIZE(...) be still that? or should it be some different length? (like, `*pcchLength`) ?

Reply:
Second argument is `cchMax` aka `The maximum number of characters allowed in psz, including the terminating null character.` -- so this is correct.